data/faq.csv

```diff
@@ -108,4 +108,4 @@ or the [Government at a Glance diagram](#partition).
 ","Pour visualiser comment les organisations et les intérêts fédéraux sont reliés entre différents systèmes d’étiquetage,
 consultez le [page des Explorer les étiquettes](#tag-explorer) ou le [diagramme Panorama de l'administration publique](#partition).
 "
-contact,Is there a way to get in contact with a person who works with GC InfoBase?,Comment communiquer avec une personne de l’InfoBase du GC?,"For any questions related to GC InfoBase, please contact the GC InfoBase mailbox at GCInfoBase@tbs-sct.gc.ca and someone on the team will reply to your question.","Pour toute question sur l’InfoBase du GC, envoyez un courriel à GCInfoBase@tbs-sct.gc.ca et un membre de l'équipe vous répondra."
\ No newline at end of file
+contact,Is there a way to get in contact with a person who works with GC InfoBase?,Comment communiquer avec une personne de l’InfoBase du GC?,"For any questions related to GC InfoBase or Treasury Board of Canada Secretariat, please visit the [contact us](#contact) page to view the different methods of contacting us. We will respond as soon as possible.","Pour toute question sur l’InfoBase du GC ou le Secrétariat du Conseil du Trésor du Canada, visitez la page [Contactez-Nous](#contact) pour des méthodes différentes de contacter. Nous repondrons dès que possible."
```

Comment:
Hmm, did someone provide this text? I don't think we should take any general questions related to TBS. Also, we're not thinking of responding to their email soooo how about something like,
`To reach us with any questions related to GC Infobase, please visit [contact us](#contact) page`?

Can't speak for French for any of these text though as I don't speak French😀